### ISSUE TITLE
Add alt text fields for project cover, background, and thumbnail images.

### ISSUE THREAD

**Issue by zdavis** [ManifoldScholar/manifold#2323]

**Comment by zdavis** [comment]
@bot bug

**Comment by zdavis** [comment]
Thank you for taking the time to report this bug. We've reviewed the issue and agree that this should be fixed. We’re adding a “planned” label to indicate that we consider this bug as part of our current work plan.

_This was an automated message, but please don't hesitate to reply. Our team watches these issues closely and will respond as soon as we're able to!_
